CONTRIBUTING.md

```diff
@@ -28,8 +28,21 @@ Our team members also monitor several other discussion forums:
 
 ## Contributing code and content
 
-We accept fixes and features! Here are some resources to help you get started on how to contribute code or new content.
-
+We are always happy to see PRs from community members both for bug fixes as well as new features.
+To help you be successful we've put together a few simple rules to follow when you prepare to contribute to our codebase:
+1. **Start from the areas we need help with, not a random one.**
```

Comment:
I would soften this a bit by removing the "not a random one".

I would also add that a sentenced along the lines of "if there is some other area not included here where you want to contribute to, first open an issue to describe the problem you are trying to solve so that we can discuss whether we think it belongs in the platform and the best approach to solve it.

Reply:
Thanks Javier. Definitely a good idea. Will incorporate this too.